<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFF6E5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="artikelLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Artikel</string>
    </property>
   </widget>
   <widget class="QLabel" name="resepLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>520</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="artikelButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EE9C20; border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/icon/Artikel_logo.png</normaloff>../img/icon/Artikel_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>1400</width>
      <height>1024</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="resepButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F75008; border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/icon/Resep_logo.png</normaloff>../img/icon/Resep_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>1400</width>
      <height>1024</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
